frankDoc/src/main/java/nl/nn/adapterframework/frankdoc/FrankDocJsonFactory.java

```diff
@@ -144,6 +145,7 @@ private JsonObject getElement(FrankElement frankElement) throws JsonException {
 		if(frankElement.isDeprecated()) {
 			result.add("deprecated", frankElement.isDeprecated());
 		}
+		addDescription(result, frankElement.getDescription());
 		addDescriptionHeader(result, frankElement.getDescriptionHeader());
```

Comment:
Voor de XSD is dit wel nodig maar ik denk dat het voor de json overtollig is zowel de header als het geheel te hebben, waar in de frontend de header uit af te leiden is.

Reply:
Gedaan.